### PR TITLE
Philosophy

### PR DESCRIPTION
**Pull request by brittany-johnson**

### CODE REVIEW

File: books/philosophy/bake [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    book: book,
+    classes: %w[read-philosopher
+                think-philosopher
+                write-philosopher
+                philo-connections
+                time-line]
+  )
+  book.notes.each do |note|
+    BakeNoteIFrames.v1(note: note)
+  end
```
Comment (kswanson33) [inline books/philosophy/bake:78]: `BakeNoteIFrames` should already be called by `BakeAutotitledNotes` -- are `iframes` unbaked if this is removed?

File: shorten [hunk 2]
```diff
@@ -27,6 +27,7 @@ case "${book}" in
   precalculus) dest="${DIR}/data/precalculus/short" && script="${DIR}/books/precalculus/shorten";;
   sociology) dest="${DIR}/data/sociology/short" && script="${DIR}/books/sociology/shorten";;
   u-physics) dest="${DIR}/data/u-physics/short" && script="${DIR}/books/u-physics/shorten";;
+  philosophy) dest="${DIR}/data/philosophy/short" && script="${DIR}/books/philosophy/shorten";;
```
Comment (kswanson33) [inline shorten:30]: i think this line & the empty shorten file can be removed if there isn't a shorten script!

**Review by kswanson33 — CHANGES_REQUESTED** [review]

File: books/philosophy/bake (hunk 1 above)
Comment (brittany-johnson) [reply books/philosophy/bake:78]: oop! didn't realize that

File: shorten (hunk 2 above)
Comment (brittany-johnson) [reply shorten:30]: oop thank you lol

File: books/philosophy/bake (hunk 1 above)
Comment (brittany-johnson) [reply books/philosophy/bake:78]: oh it looks like they are unbaked when i remove `BakeNoteIFrames`
i'll look to see why

Comment (brittany-johnson) [reply books/philosophy/bake:78]: excuse the small screen in the screenshot lol 
Screen Shot 2021-07-19 at 1 48 21 PM

i thought it was unbaked at first but it looks like there's an extra `<div class="os-has-iframe os-has-link" data-type="alternatives">` in the expected file on the left which was baked with `BakeNoteIFrames`  and `BakeAutotitledNotes` so I'm assuming thats why there is a double element

Comment (brittany-johnson) [reply books/philosophy/bake:78]: I just need to update my spec now

**Review by kswanson33 — APPROVED** [review]

**Pull request merged**